src/flint/fmpz.jl

```diff
@@ -43,7 +42,7 @@ export fmpz, FlintZZ, FlintIntegerRing, parent, show, convert, hash,
        moebius_mu, primorial, rising_factorial, number_of_partitions,
        canonical_unit, needs_parentheses, displayed_with_minus_in_front,
        show_minus_one, addeq!, mul!, isunit, isequal,
-       iszero, rand
+       iszero, rand, binomial, factorial
```

Comment:
I don't understand why you export these functions here, as they are already exported by `Base`, and it's not going to stop being that way anytime soon.

Reply:
We import them from Base and extend them. I don't recall if that is sufficient or if we need to export them again ourselves. I think this has changed over the lifetime of Julia.

Reply:
I'm certain this is not needed to re-export, and I can't recall this having changed in the lifetime of Julia. A function has multiple methods; extending it means adding one method. On the other hand, `export f` acts on the function, not the methods. So the function needs only be exported once, which Julia does for `binomial` and `factorial`.
But as this export list already contain `Base` exported functions, you might as well leave it as is, I will clean that list at once when I'm bored.